SgmlTests/Tests-Logic.cs

```diff
@@ -1,4 +1,4 @@
-﻿/*
```

Comment:
All of the low-value changes made in this file are from commit 83af0dd32da8beab9dd4aacf7f22fc2555b4bc75 where I auto-formatted this specific file as it was inconsistently formatted compared to the rest of the project.

Reply:
thanks, but in the future I prefer to move auto-reformatting to separate PR since it obscures the real code review I need to do here.